ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -166,6 +172,12 @@ export const Alleles = React.memo(({ genotype, variant, isHemiX, warning }) => (
         <Allele isAlt={genotype.numAlt > (isHemiX ? 0 : 1)} variant={variant} textAlign="right" />
         /
         {isHemiX ? '-' : <Allele isAlt={genotype.numAlt > 0} variant={variant} textAlign="left" />}
+        {genotype.mitoCn && (
+          <div>
+            &nbsp;CN:&nbsp;
+            {genotype.mitoCn.toFixed(0)}
```

Comment:
why would `mitoCn` ever be anything other than an integer? This should be returning as an integer from the pipeline/ elasticsearch, not getting post-processed here